Imath/ImathVec.h

```diff
@@ -1242,7 +1225,7 @@ Vec2<T>::lengthTiny () const
 }
 
 template <class T>
-IMATH_CONSTEXPR14 inline T
+inline T
```

Comment:
Do you know specifically why length() can't be constexpr?

Reply:
Because it calls sqrt() which is not an constexpr function.